internal/state/change_processor.go

```diff
@@ -101,8 +100,18 @@ func NewChangeProcessorImpl(cfg ChangeProcessorConfig) *ChangeProcessorImpl {
 		return gvk
 	}
 
+	processor := &ChangeProcessorImpl{
+		cfg:          cfg,
+		clusterState: clusterStore,
+	}
+
+	forceChangedByObject := func(objType client.Object, nsname types.NamespacedName) bool {
```

Comment:
does you comment apply only here or also in `store.go`?

Reply:
Both? The type declaration is better because it has a comment, but I just wonder if there's a better name. But I also can't come up with something better right now 😬

Reply:
I chose `force` to signify that it forces a change like no matter what. but I haven't come up with anything better

Reply:
Wasn't it something different in the draft PR? Like isRelevant?

Reply:
yep. But if you look at the store, which knows nothing about the graph, I don't think isRelevant is a good name there.

Reply:
Ah, ok. We can leave as is